src/Describer/InputDescriber/FormInputDescriber.php

```diff
@@ -42,12 +44,54 @@ public function describe(Input $input, Operation $operation, string $httpMethod)
         if ($input->isInQuery()) {
             $operation->parameters = array_merge($operation->parameters, $this->queryDescriber->describe($form));
         } elseif ($input->isInBody()) {
-            $operation->requestBody = new RequestBody(
-                [
-                    'required' => true,
-                    'content' => $this->bodyDescriber->describe($form, $httpMethod),
-                ]
-            );
+            $describedBody = $this->bodyDescriber->describe($form, $httpMethod);
+
+            if ($operation->requestBody === null) {
```

Comment:
We should split this method and do some early returns. It's gotten quite big.

Reply:
I've cleaned up the code, should be more readable now.